ironfish-cli/src/commands/wallet/index.ts

```diff
@@ -36,11 +36,14 @@ export class AccountsCommand extends IronfishCommand {
       return []
     }
 
+    const defaultAccount = await client.wallet.getDefaultAccount()
+    const defaultName = defaultAccount.content.account?.name
+
     ui.table(
       response.content.accounts,
       {
         name: {
-          get: (row) => row.name,
+          get: (row) => (row.name === defaultName ? `${row.name} (default)` : row.name),
```

Comment:
Does this affect the JSON output too?

Reply:
For JSON, we probably want to leave the name as is, and maybe add an extra field somewhere to point to the default account

Reply:
ie add key
```
  {
    "name": "testnet genesis",
    "id": "b0557e1c-16e1-4f2b-b491-38e87422df58",
    "head": {
      "hash": "0000004b952bb2d7e7b0cb494d444961a05a928b839b7ba7a8707bd052073000",
      "sequence": 738608,
      "inChain": true
    },
    "scanningEnabled": true,
    "viewOnly": false
  },
```